app/browser/api/ledger.js

```diff
@@ -2590,11 +2590,15 @@ const run = (state, delayTime) => {
   }
 
   if (client.isReadyToReconcile(synopsis, module.exports.onFuzzing)) {
```

Comment:
@jasonrsadler can you please move if check into the function as well

Reply:
Done